client/modules/workspace/src/AppsWizard/FormField.tsx

```diff
@@ -85,11 +186,36 @@ export const FormField: React.FC<{
         style={{ textAlign: 'left', marginBottom: description ? 0 : 16 }}
       >
         {type === 'select' ? (
-          <Select
-            {...props}
-            value={getValues(name)}
-            style={{ textAlign: 'left' }}
-          />
+          name === 'configuration.execSystemId' ? (
+            <ExtendedSelect
+              {...props}
+              name={name}
+              value={getValues(name)}
+              disabled={readOnly}
+              suffixIcon={readOnly ? null : undefined}
+              style={{
+                backgroundColor: readOnly ? '#F4F4F4' : undefined,
+              }}
+              after={SystemStatus}
+            />
+          ) : name === 'configuration.execSystemLogicalQueue' ? (
+            <ExtendedSelect
+              {...props}
+              name={name}
+              value={getValues(name)}
+              onChange={(value: string) => {
+                setValue('configuration.execSystemLogicalQueue', value);
+              }}
+              after={QueueStatus}
+            />
+          ) : (
+            <ExtendedSelect
+              {...props}
+              name={name}
+              value={getValues(name)}
+              style={{ textAlign: 'left', maxWidth: 150, width: '20%' }}
```

Comment:
This maxwidth should be removed. We want these fields to extend all the way across
![Screenshot 2025-06-03 at 4 47 56 PM](https://github.com/user-attachments/assets/9140a2b6-973b-4c82-8d3f-a5da08d1009e)